examples/src/Viewer.tsx

```diff
@@ -711,6 +724,17 @@ class Viewer extends React.Component<InputParams, ViewerState> {
         this.setState({ followObjectData: agentData });
     };
 
+    public handleCacheUpdate = (log: CacheLog) => {
+        this.setState({
+            cacheLog: log,
+            playbackPlaying: simulariumController.isPlaying(),
```

Comment:
I wouldn't think you should be updating the the playbackState when the cache is updated (ie when the app is paused I wouldn't think the cache would be updated, and then you risk the two states being out of sync)
this piece of state doesn't really seem needed in general.